### PR TITLE
Update contributions.md

### PR DESCRIPTION
**Pull request by Kaperskyguru**

### CODE REVIEW

File: en/contributions.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ Please ensure that you are sending your pull request to the correct branch and t
 ## Bug Report Checklist
 - Make sure you are using the latest released version of Phalcon before submitting a bug report. Bugs in versions older than the latest released one will not be addressed by the core team.
 - If you have found a bug, it is essential to add relevant information to reproduce it. Being able to reproduce a bug greatly reduces the time to investigate and fix it. This information should come in the form of a script, small application, or even a failing test. Please check [Submit Reproducible Test][srt] for more information.
-- As part of your report, please include additional information such as the OS, PHP version, Phalcon version, web server, memory etc.
-- If you're submitting a [Segmentation Fault][segfault] error, we would require a backtrace. Please check [Generating a Backtrace](#bug-report-generating-backtrace) for more information.
+- As part of your report, please include additional informations such as the OS, PHP version, Phalcon version, web server, memory etc.
```
Comment (niden) [inline en/contributions.md]: This should be `information`

File: en/contributions.md [hunk 2]
```diff
@@ -56,8 +56,8 @@ Please ensure that you are sending your pull request to the correct branch and t
...
+- If you're submitting a [Segmentation Fault][segfault] error, we would require a backtrace. Please check [Generating a Backtrace](#bug-report-generating-backtrace) for more informations.
```
Comment (niden) [inline en/contributions.md]: Same here

File: en/contributions.md [hunk 3]
```diff
@@ -79,19 +79,19 @@ Please follow this guides to understand how to generate the backtrace:
...
+Before submit **new functionality**, please open a [NFR](/[[language]]/[[version]]/new-feature-request) as a new issue on GitHub to discuss the impact of including the functionality or changes in the core extension. Once the functionality is approved, make sure your PR contains the following:
 
 - An update to the `CHANGELOG.md`
 - Unit Tests
 - Documentation or Usage Examples
 
 <a name='getting-support'></a>
 ## Getting Support
-If you have a question about how to use Phalcon, please see the [support page][support].
+If you have any question about how to use Phalcon, please see the [support page][support].
```
Comment (niden) [inline en/contributions.md]: Either use `a question` or `any questions`

**Review by niden — CHANGES_REQUESTED** [review]

**Review by Kaperskyguru — COMMENTED** [review]
The changes have been made.

File: en/contributions.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ Please ensure that you are sending your pull request to the correct branch and t
 - Make sure you are using the latest released version of Phalcon before submitting a bug report. Bugs in versions older than the latest released one will not be addressed by the core team.
 - If you have found a bug, it is essential to add relevant information to reproduce it. Being able to reproduce a bug greatly reduces the time to investigate and fix it. This information should come in the form of a script, small application, or even a failing test. Please check [Submit Reproducible Test][srt] for more information.
 - As part of your report, please include additional information such as the OS, PHP version, Phalcon version, web server, memory etc.
-- If you're submitting a [Segmentation Fault][segfault] error, we would require a backtrace. Please check [Generating a Backtrace](#bug-report-generating-backtrace) for more information.
+- If you're submitting a [Segmentation Fault][segfault] error, we would require a backtrace. Please check [Generating a Backtrace](#bug-report-generating-backtrace) for more informations.
```
Comment (niden) [inline en/contributions.md]: This needs to be `information`

**Review by niden — CHANGES_REQUESTED** [review]

File: en/contributions.md (hunk 4 above)
Comment (Kaperskyguru) [reply en/contributions.md]: Ok... Thanks for your patience

**Review by Kaperskyguru — COMMENTED** [review]
Done!

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you!!

**Comment by Kaperskyguru** [comment]
Wow...  That was my first contribution to project and my first PR too...  Thanks for your assistance.

**Comment by niden** [comment]
Thank you for the contribution! Keep up the good work!

**Pull request merged**